Library/Styles/LoginStyles.swift

```diff
@@ -100,6 +100,17 @@ public let loginRootStackViewStyle =
         : .init(topBottom: Styles.grid(2), leftRight: Styles.grid(3))
 }
 
+public let facebookConfirmationStackViewStyle =
+  UIStackView.lens.layoutMarginsRelativeArrangement .~ true
+    <> UIStackView.lens.layoutMargins %~~ { _, stack in
```

Comment:
Thank you @stephencelis! I like it much more than new style. I will make the change and will resubmit..